src/org/traccar/api/AsyncSocket.java

```diff
@@ -45,7 +47,9 @@ public AsyncSocket(long userId) {
     public void onWebSocketConnect(Session session) {
         super.onWebSocketConnect(session);
 
-        sendData(KEY_POSITIONS, Context.getConnectionManager().getInitialState(userId));
+        Map<String, Collection<?>> data = new HashMap<String, Collection<?>>();
```

Comment:
Traccar targets Java 7 and higher, so you can use diamond operator on the right hand side.